meson.build

```diff
@@ -231,10 +231,29 @@ else
   lvfs_metadata_format = 'gz'
 endif
 conf.set_quoted('FU_LVFS_METADATA_FORMAT', lvfs_metadata_format)
-gusb = dependency('gusb', version: '>= 0.4.8', fallback: ['gusb', 'gusb_dep'], required: get_option('gusb'))
-if gusb.found()
-  conf.set('HAVE_GUSB', '1')
+
+# FreeBSD is missing some libusb symbols
+libusb = dependency('libusb-1.0', version : '>= 1.0.9', required: get_option('usb'))
+if libusb.found()
+  conf.set('HAVE_LIBUSB', '1')
+  conf.set_quoted('LIBUSB_VERSION', libusb.version())
```

Comment:
we probably want to export this in fwupdmgr get-versions/fwupdtool get-versions etc.

Reply:
Yup, this is here to get the version out of the pkgconfig file in to the header so we can have:

    [hughsie@hughsie-work-lan build (hughsie/no-libgusb *%)]$ ./src/fwupdmgr --version
    compile   info.libusb                   1.0.27
    ...